artemis-jdbc-store/src/main/java/org/apache/activemq/artemis/jdbc/store/drivers/AbstractJDBCDriver.java

```diff
@@ -109,7 +109,7 @@ public void stop() throws SQLException {
    protected abstract void createSchema() throws SQLException;
 
    protected final void createTable(String... schemaSqls) throws SQLException {
-      createTableIfNotExists(connection, sqlProvider.getTableName(), schemaSqls);
+      createTableIfNotExists(sqlProvider.getTableName(), schemaSqls);
```

Comment:
easier to ask than reading the code :)


Why removing this parameter here?


(sorry if I'm lazy.. I have a bunch of other PRs to check.. so it was easier to ask).

Reply:
`createTableIfNotExists` was `private static` before. I needed to change it to non-static in order for `TestJDBCDriver` to be able to access the underlying connection to create the empty table for the test.